cmd/gen-go-sample/testdata/sample_unary.want

```diff
@@ -36,6 +38,33 @@ func sampleUnaryMethod(theX int64, theB string, theFoo string) error {
 	// theX := 42
 	// theB := "foobar"
 	// theFoo := "myfoo"
+	// fileArg := "path/to/local/file/arg.txt"
+
+	var file io.Reader
+	var err error
+
+	file, err = os.Open("path/to/local/file/default.txt")
+	if err != nil {
+		log.Fatalf("Failed to read file: %v", err)
+	}
+	defer file.Close()
+	var fileDefaultContent []byte
+	fileDefaultContent, err = ioutil.ReadAll(file)
+	if err != nil {
+		log.Fatalf("Failed to read file: %v", err)
+	}
+
+	file, err = os.Open(fileArg)
```

Comment:
I'd like to use this:

```go
fileBytes, err := ioutil.ReadFile(file)
if err != nil {
    return err
}
```

Examples:
 - https://github.com/GoogleCloudPlatform/golang-samples/blob/master/videointelligence/video_analyze/video_analyze.go#L37-L40
 - https://github.com/GoogleCloudPlatform/golang-samples/blob/master/speech/caption/caption.go#L107-L110 (uses data)

We use that simple `ReadFile` everywhere I can find

Reply:
Sounds good!